### PR TITLE
Object

### PR DESCRIPTION
**Pull request by Kartynnikk**

### CODE REVIEW

File: object.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
+// Создайте объект calculator (калькулятор) с тремя методами:
+
+// read() (читать) запрашивает два значения и сохраняет их как свойства объекта.
+// sum() (суммировать) возвращает сумму сохранённых значений.
+// mul() (умножить) перемножает сохранённые значения и возвращает результат.
+
+let calculator = {
+    read() {
+        this.a = +prompt("Введите значение а");
```
Comment (kathrynowy) [inline object.js:9]: what if I enter 'abc'?

File: object.js [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+
+let teapot = {
+    color: "white",
+    status: "isOn",
+
+    get() {
+        this.time = +prompt("Сколько лет работал чайник?", 6);
+    },
+
```
Comment (kathrynowy) [inline object.js:64]: не сделано

должны быть поля , которые я описала в обьекте

**Review by kathrynowy — CHANGES_REQUESTED** [review]

File: object.js (hunk 1 above)
Comment (Kartynnikk) [reply object.js:9]: it will be Nan

Comment (kathrynowy) [reply object.js:9]: i know))))
you should avert using NaN further on
check the value to be true

File: object.js [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+    rate: 5,
+    workdays: 22,
+    getCurrentSalary() {
+        return this.rate * this.workdays;
+    }
+};
+
+const worker = employee;
+
+worker.firstname = "Ira", "Petrova", 5, 25;
```
Comment (kathrynowy) [inline object.js:115]: what is it?

Comment (kathrynowy) [reply object.js:115]: you should invoke a function

Comment (kathrynowy) [reply object.js:115]: employee("Ira", "Petrova", 5, 25);

**Review by kathrynowy — CHANGES_REQUESTED** [review]

File: object.js (hunk 1 above)
Comment (Kartynnikk) [reply object.js:9]: Maybe we need to add the condition with if(this.a && this b){ return this.a*this.b}
The result will be underfined

File: object.js (hunk 2 above)
Comment (Kartynnikk) [reply object.js:76]: edited

File: object.js (hunk 3 above)
Comment (Kartynnikk) [reply object.js:115]: edited

File: object.js (hunk 1 above)
Comment (kathrynowy) [reply object.js:9]: check if it's a number then do smth 
If not - add zero value to the variable

**Review by kathrynowy — APPROVED** [review]

File: object.js (hunk 1 above)
Comment (Kartynnikk) [reply object.js:9]: edited

**Pull request merged**